vnext/ReactUWP/Views/ShadowNodeBase.cpp

```diff
@@ -154,5 +154,10 @@ void ShadowNodeBase::EnsureHandledKeyboardEventHandler() {
   }
 }
 
+void ShadowNodeBase::YellowBox(const std::string &message) {
```

Comment:
>) [](start = 57, length = 1)

Can we mark it as noexcept?

Reply:
@vmoroz I don't think we can guarantee that callJSFunction doesn't throw, can we?